docs/app/Examples/elements/Segment/SegmentExamples.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import SegmentTypesExamples from './Types/SegmentTypesExamples';
+import SegmentGroupsExamples from './Groups/SegmentGroupsExamples';
+// import SegmentStatesExamples from './Types/SegmentStatesExamples';
+// import SegmentVariationsExamples from './Variations/SegmentVariationsExamples';
```

Comment:
There's still a little work to be done here, but at this point I'd like to get the introduction of the `Segments` component nailed down before continuing.